pineappl_cli/src/plot.py

```diff
@@ -71,24 +85,31 @@
 
 
 def main():
-    panels = [
-        {enable_int}plot_int,
-        {enable_abs}plot_abs,
-        {enable_rel_ewonoff}plot_rel_ewonoff,
-        {enable_abs_pdfs}plot_abs_pdfs,
-        {enable_ratio_pdf}plot_ratio_pdf,
-        {enable_double_ratio_pdf}plot_double_ratio_pdf,
-        {enable_rel_pdfunc}plot_rel_pdfunc,
-        {enable_rel_pdfpull}plot_rel_pdfpull,
-    ]
+    panels = []
+    if enable_int:
```

Comment:
All of these `if`s should be put together in a loop.

Reply:
[see above](https://github.com/NNPDF/pineappl/pull/312#discussion_r1768479509)

Reply:
Done in commit 24b97b8cb46fdf5511a94219e7e40e53f74217af.

Reply:
Improved in 143b5d53 - `globals()` should be avoided if possible ...

Reply:
I believe `globals()` is OK since execute a generated script, but if we want to split parts of it into a library this is better.